example/server.js

```diff
@@ -5,6 +5,6 @@ webpack.listen(8080, "localhost", function() {
   console.log("Server is listening on http://localhost:8080")
 });
 
-api.listen(7000, function() {
-  console.log('API listening at http://localhost:7000');
+api.listen(7654, function() {
```

Comment:
Hmm. In that case, if it's going to be random anyway, what about setting up a `port` variable and just incrementing it by one for the api?